specification/billing/resource-manager/Microsoft.Billing/privatepreview/2020-11-01-privatepreview/examples/BillingActivityList.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "api-version": "2020-11-01-privatepreview",
+    "billingAccountName": "7303440",
+    "startDate": "{2019-01-10}",
+    "endDate": "{2020-02-10}",
+    "activityLogEventCategory": "ManageAccountAdd"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/7303440/billingActivityList",
```

Comment:
please update the type in examples too - you will likely have automated checks failing since they mismatch